Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs

```diff
@@ -0,0 +1,1128 @@
+                                    
+                                    if (busVariables.Keys.Contains(linkedVariable)==false)
+                                    {
+                                        busVariables.Add(linkedVariable, valueParts[2]);
+                                    }
+                                }
+                            }
+
+                            //determine if the param is Flow Control Param or not based on it value and agreed sign "$$_"
+                            if (paramSelectedValue.StartsWith("$$_"))
+                            {
+                                isflowControlParam = false;
+                                if (paramSelectedValue.StartsWith("$$_"))
+                                    paramSelectedValue = paramSelectedValue.Substring(3);//get value without "$$_"
+                            }
+                            else if (paramSelectedValue != "<Empty>")
+                                isflowControlParam = true;
+
+                            //check if already exist param with that name
+                            VariableBase stepActivityVar = stepActivity.Variables.Where(x => x.Name.ToUpper() == param.ToUpper()).FirstOrDefault();
+                            if (stepActivityVar == null)
+                            {
+                                //#Param not exist so add it
+                                if (isflowControlParam == true)
+                                {
+                                    //add it as selection list param                               
+                                    stepActivityVar = new VariableSelectionList();
+                                    stepActivityVar.Name = param;
+                                    stepActivity.AddVariable(stepActivityVar);
+                                    stepActivity.AutomationStatus = eActivityAutomationStatus.Development;//reset status because new flow control param was added
+                                }
+                                else
+                                {
+                                    //add as String param
+                                    stepActivityVar = new VariableString();
+                                    stepActivityVar.Name = param;
+                                    ((VariableString)stepActivityVar).InitialStringValue = paramSelectedValue;
+                                    stepActivity.AddVariable(stepActivityVar);
+                                }
+                            }
+                            else
+                            {
+                                //#param exist
+                                if (isflowControlParam == true)
+                                {
+                                    if (!(stepActivityVar is VariableSelectionList))
+                                    {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)